packages/ringcentral-widgets-test/test/integration-test/AllCallsPage/helper.js

```diff
@@ -0,0 +1,101 @@
+import * as mock from 'ringcentral-integration/integration-test/mock';
+import subscriptionBody from 'ringcentral-integration/integration-test/mock/data/subscription.json';
+import deviceBody from '../calls/data/device.json';
+import {
+  mockActiveCalls,
+  getInboundCall,
+  mockDetailedPresencePubnub,
+  makeCall,
+  mockGeneratePresenceApi,
+  mockGeneratePresenceUpdateApi,
+  mockGenerateActiveCallsApi,
+} from '../../support/callHelper';
+import { timeout } from '../shared';
+import pubnubPresence from './data/pubnubPresence';
```

Comment:
there is a function mockDetailedPresencePubnub in callHelper.js which is used to generate pubnubPresence data, you can try to use this funciton but not import other static data